i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md

```diff
@@ -218,166 +333,1027 @@ Ce connecteur de supervision est compatible avec n'importe quelle distribution L
 
-1. Installer le Plugin sur tous les collecteurs Centreon supervisant des ressources Linux SSH :
+Utilisez les commandes ci-dessous en fonction du gestionnaire de paquets de votre système d'exploitation :
+
+<Tabs groupId="sync">
+<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
 
 ```bash
-yum install centreon-plugin-Operatingsystems-Linux-Ssh.noarch
+dnf install centreon-plugin-Operatingsystems-Linux-Ssh
 ```
 
-2. Sur l'interface Web de Centreon, installer le connecteur de supervision *Linux SSH* depuis la page **Configuration > Gestionnaire de connecteurs de supervision**
-
 </TabItem>
-<TabItem value="Offline License" label="Offline License">
-
-1. Installer le Plugin sur tous les collecteurs Centreon supervisant des ressources Linux SSH :
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
 
 ```bash
-yum install centreon-plugin-Operatingsystems-Linux-Ssh.noarch
+dnf install centreon-plugin-Operatingsystems-Linux-Ssh
 ```
 
-2. Sur le serveur Central Centreon, installer le connecteur de supervision via le RPM:
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
 
 ```bash
-yum install centreon-pack-operatingsystems-linux-ssh.noarch
+apt install centreon-plugin-operatingsystems-linux-ssh
 ```
 
-3. Sur l'interface Web de Centreon, installer le connecteur de supervision *Linux SSH* depuis la page **Configuration > Gestionnaire de connecteurs de supervision**
+</TabItem>
+<TabItem value="CentOS 7" label="CentOS 7">
+
+```bash
+yum install centreon-plugin-Operatingsystems-Linux-Ssh
+```
 
 </TabItem>
 </Tabs>
 
-## Configuration
+## Utiliser le connecteur de supervision
+
+### Utiliser un modèle d'hôte issu du connecteur
+
+1. Ajoutez un hôte à Centreon depuis la page **Configuration > Hôtes**.
+2. Complétez les champs **Nom**, **Alias** & **IP Address/DNS** correspondant à votre ressource.
+3. Appliquez le modèle d'hôte **OS-Linux-SSH-custom**. Une liste de macros apparaît. Les macros vous permettent de définir comment le connecteur se connectera à la ressource, ainsi que de personnaliser le comportement du connecteur.
+4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires.
 
-Créer un nouvel hôte dans Centreon et lui appliquer le modèle d'hôte "OS-Linux-SSH-custom".
-Une fois le modèle défini, vous devez définir des valeurs en fonction du backend SSH choisi.
-3 backends SSH sont disponibles pour se connecter au serveur Linux : *sshcli*, *plink* et *libssh* qui sont détaillés ci-dessous.  
+| Macro           | Description                                                                                                                                                         | Valeur par défaut | Obligatoire |
+|:----------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
```

Comment:
no mandatory options? contradicts https://docs.centreon.com/fr/pp/integrations/plugin-packs/procedures/operatingsystems-linux-ssh/#configuration
Also, missing info?

Reply:
if the user is named centreon-engine and the key exchange has been done, no parameter is mandatory.